saleor/graphql/account/mutations.py

```diff
@@ -649,7 +625,7 @@ def mutate(cls, root, info, id, type):
             raise ValueError('Unknown value of AddressTypeEnum: %s' % type)
```

Comment:
It's not changed in this PR, but isn't it validated by GraphQL itself? When you try to type query/mutation with enum values not defined in schema (in this case from here https://github.com/mirumee/saleor/blob/23208ede820d720645fe6bf2e0a5e4cda0b6f794/saleor/checkout/__init__.py) graphql should disallow process such request.